pool/web/handler/admin_experiments.ml

```diff
@@ -411,4 +385,22 @@ end = struct
       |> Guardian.validate_generic
     ;;
   end
+
+  let search =
```

Comment:
If there are multiple urls to search experiments, why not split this up in multiple middlewares? That would be easier to understand.

Reply:
As there is only one route, please cleanup the access middleware.

Reply:
done